### PR TITLE
sw: Add support for multiple yaml files in run.py

### PR DESCRIPTION
**Pull request by JosseVanDelm**
Small tweak which is useful for #57 , that splits run.yaml into multiple different yaml files.
I just made the argument a "one or more" and put a for loop around the test readout logic.

### CODE REVIEW

File: util/sim/sim_utils.py [hunk 1]
```diff
@@ -79,18 +80,21 @@ def resolve_relative_path(base_path, s):
...
+    all_tests = []
+    for testlist in testlists:
+        testlist_path = Path(testlist).absolute()
+        with open(testlist_path, 'r') as f:
+            tests = yaml.safe_load(f)['runs']
+        # Convert relative paths in testlist file to absolute paths
+        for test in tests:
+            test['elf'] = testlist_path.parent / test['elf']
+            if 'cmd' in test:
+                test['cmd'] = [resolve_relative_path(testlist_path.parent, arg) for arg in test['cmd']]
```
Comment (rgantonio) [inline util/sim/sim_utils.py:94]: break into lines? haha flake8 just complains...
```suggestion
                test['cmd'] = [resolve_relative_path(testlist_path.parent, arg)\
                 for arg in test['cmd']]
```

**Review by rgantonio — APPROVED** [review]
EZPZ, just resolving flake8 complaint

File: util/sim/sim_utils.py (hunk 1 above)
Comment (JosseVanDelm) [reply util/sim/sim_utils.py]: Yeah chatgpt helped me a bit thanks!

**Comment by JosseVanDelm** [comment]
Seems to be working on banshee, so I'm going ahead already

**Pull request merged**